src/symtab2gb/symtab2gb_parse_options.h

```diff
@@ -17,6 +17,7 @@ Author: Diffblue Ltd.
 
 #define SYMTAB2GB_OPTIONS                                                      \
   "(" SYMTAB2GB_OUT_FILE_OPT "):"                                              \
+  "(verbosity):"
```

Comment:
Nitpick - Missing trailing `\` which should be there for merge-conflict / diff-minimisation related reasons.

Reply:
(And I haven't found an option to teach clang-format otherwise.)

Reply:
We turn off clang-format in the other *_parse_options.h

Reply:
That was actually also the case here, my bad. I've added the backslash.